hms-app/src/common/constants.ts

```diff
@@ -1,5 +1,6 @@
 import logo from '../assets/idealo-logo.png'
 
 export const NULL_DATE = new Date('1970-01-01')
-export const VALID_DATE = new Date('2000-01-01')
+export const MIN_DATE = new Date('1971-01-01')
+export const MAX_DATE = new Date('2039-12-31')
```

Comment:
why is this the "max" date?

Reply:
at the very least comments explaining the rationale behind all of these three constants would be very helpful

Reply:
and since the IdeationPortal hackathon exists in the backend it cannot have a null date 🤦

ok yeah I get it all now, the `MIN_DATE` is for sure a hack but makes sense

Reply:
yeah bit hacky but for now the best ways I would say